datasets/ua/war_sanctions/crawler.py

```diff
@@ -423,6 +429,43 @@ def emit_relation(
     return other
 
 
+def emit_ownership_chain(context, legal_entity, rostec_ownership):
+    """
+    Creates and emits the ownership chain for a series of entities.
+
+    The first element in the rostec_ownership list is the initial legal entity,
+    and the last element is Rostec. Intermediate elements represent the companies
+    in between the chain. This function sets up the ownership hierarchy.
+    (e.g. ['JSC ODK-KLIMOV', 'JSC UEC', 'ROSTEC'])
+    Full ownership structure is available at: https://war-sanctions.gur.gov.ua/en/rostec
+
+    Args:
+        context: Context
+        legal_entity: The initial legal entity (the first in the ownership chain).
+        rostec_ownership: A list of entities that form the ownership chain, from the legal entity to Rostec.
+    """
+    # `asset` refers to the previous entity in the chain, starting with the first entity
+    asset = legal_entity
+
+    # Iterate over the remaining entities in the rostec_ownership
+    for entity in rostec_ownership[1:]:
```

Comment:
Since it's so common in our code that `entity` is an `Entity` type, perhaps we can have a name that's a bit different from that here

```suggestion
    for abbrev_name in rostec_ownership[1:]:
```